google_nest_sdm/webrtc_util.py

```diff
@@ -0,0 +1,114 @@
+# SDP Direction Constants
+DIRECTION_SENDRECV = 'sendrecv'
+DIRECTION_SENDONLY = 'sendonly'
+DIRECTION_RECVONLY = 'recvonly'
+DIRECTION_INACTIVE = 'inactive'
+
+AVAILABLE_DIRECTIONS = [DIRECTION_SENDRECV,
```

Comment:
Let's use an enum here:

```
class AvailableDirections(StrEnum):
    SENDRECV = 'sendrecv'
    SENDONLY = 'sendonly'
    RECVONLY = 'recvonly'
    INACTIVe = 'inactive'
```

Then you can do do `for direction in AvailableDirections` without tracking it separately.